changelog.adoc

```diff
@@ -9,6 +9,7 @@ and this project tries to adhere to https://semver.org/spec/v2.0.0.html[Semantic
 
 === fixes
 
+* https://github.com/docToolchain/docToolchain/issues/1150[#1150 generateDeck with Java 17 shows WARN FilenoUtil]
```

Comment:
I would not add this to the release notes. Since we switched in this Release to Java 17 this bug report never hit the user.
Maybe we should add a different label for bugs we find in our `latest` releases.

Reply:
I removed the changelog entry.